web/styles/input_pill.css

```diff
@@ -178,13 +179,26 @@
             color: inherit;
         }
 
-        > .pill-value {
-            padding: 5px;
+        > .pill-label {
             max-width: 165px;
         }
     }
 }
 
+/* Specific styling, on top of shared styling above,
+to vertically align the pill with surrounding text. */
+.stream_creator_details > .pill-container {
```

Comment:
This could maybe benefit from an `inline-with-text-pill` more declarative class, perhaps?

Reply:
Added new css class `.inline_with_text_pill` for this. We seem to be preferring `_` instead of `-` in new class names, right?

<details><summary>git grep on new class</summary>
<p>

```css
(zulip-py3-venv) vagrant@fbc57b970081:/srv/zulip$ git grep ".inline_with_text_pill"
web/styles/input_pill.css:.stream_creator_details > .pill-container-btn.inline_with_text_pill {
web/templates/user_display_only_pill.hbs:<span class="pill-container display_only_pill {{#if is_inline}}pill-container-btn inline_with_text_pill{{/if}}">
```
</p>
</details>

Reply:
https://zulip.readthedocs.io/en/stable/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids covers this -- we're very slowly moving towards `-`s, but `_`s used to be the default. Not particularly important one way or the other, but FYI.